src/figuresClasses.ts

```diff
@@ -1,19 +1,68 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
```

Comment:
Would be better to use `enum` data type here 
And then you don't need to write like so `public shape: 'triangle' = 'triangle'` in the classes. You can just use a specific field like `public shape: Shape.Triangle` etc